src/RoslynPad.Editor.Windows.Test/RoslynPad.Editor.Windows.Test.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="NUnit" Version="3.10.1" />
```

Comment:
I'd rather use xunit. Would it be much trouble to ask you to switch?

Reply:
I've put a few more details on #208 - I can certainly take another look at XUnit though.